importexport/capella/capella.cpp

```diff
@@ -494,14 +494,14 @@ static bool findChordRests(BasicDrawObj const* const o, Score* score, const int
       for (Segment* seg = score->tick2segment(tick); seg; seg = seg->next1()) {
             if (seg->segmentType() != SegmentType::ChordRest)
                   continue;
-            ChordRest* cr = static_cast<ChordRest*>(seg->element(track));
+            ChordRest* cr = toChordRest(seg->element(track));
             if (cr) {
-                  if (graceNumber1 > 0) { // the spanner is starting from a grace note
-                        Chord* chord = static_cast<Chord*>(cr);
-                        foreach(Chord* cc, chord->graceNotes()) {
+                  if (graceNumber1 > 0 && cr->isChord()) { // the spanner is starting from a grace note
```

Comment:
This additional check is the core of this fix. The other changes are more or less cosmetic